### ISSUE TITLE
Possibility to open Recycle app from an item of the sidebar

### ISSUE THREAD

**Issue by bart-at-mobicage** [our-city-app/oca-backend#786]
- [ ] In Rogerthat Control Center make it possible to define opening an app using the sidebar 
- [ ] Implement in iOS (don't forget `LSApplicationQueriesSchemes`)
- [ ] Implement in Android

**Comment by RubenMattan** [comment]
@bart-at-mobicage In case the app is not installed do you want to open the play/app store?

**Comment by bart-at-mobicage** [comment]
yes

**Comment by RubenMattan** [comment]
android: https://github.com/rogerthat-platform/rogerthat-android-client/pull/341/commits/070d663baad376d4b09afeec9df31534a2437c6f

**Comment by RubenMattan** [comment]
ios (without LSApplicationQueriesSchemes): https://github.com/rogerthat-platform/rogerthat-ios-client/pull/212/commits/17320611e4e596d86eadadb194db29bd14d14d3a

**Comment by RubenMattan** [comment]
@abom can you implement the control center?

**Comment by RubenMattan** [comment]
So params on a navigation item with action 'open_app' contain the following:
- android_app_id: **mobi.inthepocket.fostplus.recyclage** (https://play.google.com/store/apps/details?id=mobi.inthepocket.fostplus.recyclage&hl=en)
- android_scheme: **recycle://xxxx**
- ios_app_id: **id730904895** (https://itunes.apple.com/be/app/recycle!/id730904895?mt=8)
- ios_scheme: **recycle://xxxx**

these values need to be filled in manually

**Comment by bart-at-mobicage** [comment]
https://itunes.apple.com/app/id730904895 works too.

**Comment by abom** [comment]
control center: https://github.com/rogerthat-platform/plugin-rogerthat-control-center/pull/267

I added open_app as an action (not an action type), i cannot find any action to associate open_app to it as a type.

**Comment by RubenMattan** [comment]
I don't think this will break anything however i thought about it some more and would like that the **action_type** is '**open**' and **action** '**app**'.
That way we can add more stuff later on that needs to be opened like a website of whatever.

Abom before you do anything @bart-at-mobicage what is your opinion?

**Comment by bart-at-mobicage** [comment]
This is also good for me.

**Comment by RubenMattan** [comment]
Implementation is done. However we need to wait until recycle adds a scheme to open ther iOS app
